### PR TITLE
Search

### PR DESCRIPTION
**Pull request by kmharishgowda**
Updated Seach page, Pls approve PR

### CODE REVIEW

File: src/com/pages/Searchpage.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.pages;
+
+public class Supportpage {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		System.out.println("Support page");
```
Comment (HarishGowda27) [inline src/com/pages/Searchpage.java:7]: update to search page to print

**Review by HarishGowda27 — CHANGES_REQUESTED** [review]
Need changes but it's execptional

File: src/com/pages/Searchpage.java (hunk 1 above)
Comment (HarishGowda27) [reply src/com/pages/Searchpage.java:7]: done

**Review by HarishGowda27 — APPROVED** [review]
Approved PR

**Comment by kmharishgowda** [comment]
Merging the pull request

**Pull request merged**